src/client/opamArg.ml

```diff
@@ -209,6 +209,10 @@ let environment_variables =
        'download-command' value from the main config file.";
       "NOCHECKSUMS", cli_original, (fun v -> NOCHECKSUMS (env_bool v)),
       "enables option --no-checksums when available.";
+      "REPOSITORYTARRING", cli_from cli2_2,
+      (fun b -> REPOSITORYTARRING (env_bool b)),
+      "if set, enable repository tarring in temporary directory \
+       optimisation.";
```

Comment:
```suggestion
      "internally store the repositories as tar.gz files. This can be much \
       faster on filesystems that don't cope well with scanning large trees \
       but have good caching in /tmp.";
```

Reply:
maybe adding "but is slower in the general case" at the end would be helpful too i think

Reply:
If you wish! `but have good caching in /tmp` let suppose that if it's not the case you won't get a speedup but not that it'll be a big slowdown indeed :)